qemu/tests/block_resize.py

```diff
@@ -6,6 +6,165 @@
 from virttest import utils_test
 from virttest import storage
 from virttest import data_dir
+from virttest.utils_windows import drive
+
+
+def get_block_size(test, session, block_cmd, block_pattern):
+    """
+    Get the disk size inside guest.
+
+    param block_cmd: run block_cmd command to get detail disk size info.
+    param block_pattern: use this pattern to filter the needed.
+
+    param return: return current block size.
```

Comment:
No need blank line here.

Reply:
return is not paramete, and the normal docstring usage is:

:param block_cmd: xxx
:return: xxx